examples/get-started/src/App.module.css

```diff
@@ -0,0 +1,46 @@
+article {
+  display: grid;
+  gap: 2rem;
+  grid-template:
+    'Header Toggle'
+    'Sections Sections';
```

Comment:
interesting name lol

Reply:
Welcome to the world of grid templates. This syntax says that we'll have 2 rows and 2 columns and that the "Sections" named grid region will span 2 columns. The `grid-area: Sections` in the `.sections` class tells it that it should plop into this place.